packages/devtools_app/lib/src/framework/notifications_view.dart

```diff
@@ -100,7 +57,7 @@ class _NotificationsState extends State<_Notifications> with AutoDisposeMixin {
   OverlayEntry? _overlayEntry;
 
   final List<_Notification> _notifications = [];
-  late NotificationController controller;
+  late NotificationService controller;
```

Comment:
nit: rename to service or notificationsService

Reply:
removed it